app/connectors/PdfGeneratorServiceConnector.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import javax.inject.{Inject, Singleton}
+import play.api.libs.ws.{WSClient, WSResponse}
```

Comment:
We have to use `uk.gov.hmrc.play.bootstrap.http.HttpClient` rather than `WSClient`

Reply:
you need to use `http-verbs` - see the other connectors